waspc/data/Generator/templates/server/src/config.js

```diff
@@ -13,25 +13,44 @@ const config = {
     env,
     port: parseInt(process.env.PORT) || 3001,
     databaseUrl: process.env.DATABASE_URL,
+    trustProxyCount: undefined,
     {=# isAuthEnabled =}
-    auth: {
-      jwtSecret: undefined
-    }
+    session: {
+      cookie: {
+        name: process.env.SESSION_COOKIE_NAME || 'wasp_session',
+        secret: undefined,
+        maxAge: parseInt(process.env.SESSION_COOKIE_MAX_AGE) || 7 * 24 * 60 * 60 * 1000, // ms
+      },
+    },
+    csrf: {
+      cookie: {
+        name: process.env.CSRF_COOKIE_NAME || 'wasp_csrf',
+      },
+    },
     {=/ isAuthEnabled =}
+    frontendUrl: undefined,
   },
   development: {
+    trustProxyCount: parseInt(process.env.TRUST_PROXY_COUNT) || 0,
     {=# isAuthEnabled =}
-    auth: {
-      jwtSecret: 'DEVJWTSECRET'
-    }
+    session: {
+      cookie: {
+        secret: process.env.SESSION_COOKIE_SECRET || 'sessionSecret',
+      },
+    },
     {=/ isAuthEnabled =}
+    frontendUrl: process.env.REACT_APP_URL || 'http://localhost:3000',
   },
   production: {
+    trustProxyCount: parseInt(process.env.TRUST_PROXY_COUNT) || 1,
     {=# isAuthEnabled =}
-    auth: {
-      jwtSecret: process.env.JWT_SECRET
-    }
+    session: {
+      cookie: {
+        secret: process.env.SESSION_COOKIE_SECRET,
```

Comment:
Should we run some checks on the secret here, to ensure they provide something long / complex enough? If not now, we could put a TODO for it. I think we had smth similar for JWT secret -> hm actually I think we had no check but we had a piece in docs describing how to generate it? Actual check sounds even better. Maybe they check it in `session` library?

Reply:
I do not think there are any default checks, I could add one in our session library though if we want.

Reply:
I added a check helper function `checkCookieSecretLength` in config.js that is used in session.js now 👍🏻